Feature/Room.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Validating Rooms are displayed when date is input
+
+  Scenario Outline: The rooms are visible when user inputs date
+    Given The user is at the home page
+    When The user clicks on the "Rooms" top menu
+    And The user clicks on the Click to Select Dates tab
```

Comment:
please research term before using them. this is a field not a tab

Reply:
Done